### DISCUSSION TITLE
IEEE 802.15.4 implementation in Zephyr

### DISCUSSION THREAD

**Discussion by jchellap** [zephyrproject-rtos/zephyr#70022] [Q&A]
Hi All,
The 15.4 implementation in Zephyr is complete requirements of IEEE 802.15.4 2020 standards for open thread and zigbee?
Kindly guide, this is required for our internal planning.
Thanks

**Comment by ndrs-pst** [comment]
@jchellap According to this [link](https://docs.zephyrproject.org/3.6.0/connectivity/networking/api/ieee802154.html), it states, `We’re currently following the IEEE 802.15.4-2020 specification.`
Therefore, this does not necessarily imply that the implementation is complete, but at least it is in compliance.
